### PR TITLE
Add deploy action

### PR DESCRIPTION
**Pull request by jakep36**
switch to `npm run build`

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -12,11 +12,8 @@ jobs:
     steps:
         - uses: actions/checkout@v2
```
Comment (Tr3yb0) [inline .github/workflows/main.yml:14]: Do you need npm ci as well?

**Review by jakep36 — COMMENTED** [review]
test

**Review by Tr3yb0 — COMMENTED** [review]
reviewed

**Review by Tr3yb0 — APPROVED** [review]
approved

**Pull request merged**